listings/templates/listings/gathering_centers/detail.html

```diff
@@ -13,10 +13,27 @@
                 <div class="col-lg-8 col-lg-offset-2 text-center">
 
                     <h1>{{ object.city }} - {% if object.location_name %}{{ object.location_name }}{% else %}{{ object.address }}{% endif %}</h1>
+                    {% if object.hours %}
+                    <p>{{ object.hours }}</p>
+                    {% endif %}
                     {% if object.geoposition %}
                         {% leaflet_map "main" callback="main_map_init" %}
                     {% endif %}
                 </div>
+                <div class="col-lg-8 col-lg-offset-2">
+                    {% if object.most_needed %}
+                    <h2>Most Needed</h2>
```

Comment:
I think we should wrap all literal strings in trans tags right away. The first sight will be in Slovenian. ;-)